README.md

```diff
@@ -1,10 +1,13 @@
-[![Build Status](https://travis-ci.org/lightblue-platform/lightblue-core.svg?branch=master)](https://travis-ci.org/lightblue-platform/lightblue-core) [![Coverage Status](https://coveralls.io/repos/lightblue-platform/lightblue-core/badge.png?branch=master)](https://coveralls.io/r/lightblue-platform/lightblue-core?branch=master) [![Stories in Ready](https://badge.waffle.io/lightblue-platform/lightblue.svg?label=ready&title=Ready)](http://waffle.io/lightblue-platform/lightblue)
```

Comment:
Nice, I noticed that you referenced the lightblue parent projects (that have the other ones as maven modules) as it has the overall coverage metrics. 

Another point that I saw it it used to reference our waffle account, do you think it doesn't have to be referenced there anymore? I thought that link could be helpful so see a new feature on roadmap or maybe a bug.